Sources/NIOHPACK/StringRing.swift

```diff
@@ -229,7 +232,7 @@ struct StringRing {
         }
         
         // single write, should be straightforward
-        self._storage.withVeryUnsafeBytes { ptr in
+        self.withVeryUnsafeBytes { ptr in
```

Comment:
couldn't you use `withUnsafeReadableBytes` here given that you mark everything as readbale?

Reply:
I could. Does it matter though?

Reply:
no